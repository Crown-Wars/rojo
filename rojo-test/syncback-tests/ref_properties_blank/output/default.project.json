{
  "name": "ref_properties_blank",
  "tree": {
    "$className": "DataModel",
    "Workspace": {
      "$path": "src"
    }
  }
}