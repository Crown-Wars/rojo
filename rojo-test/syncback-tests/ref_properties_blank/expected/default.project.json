{
  "name": "ref_properties_blank",
  "tree": {
    "$className": "DataModel",
    "Workspace": {
      "$properties": {
        "CSGAsyncDynamicCollision": {
          "Enum": 0
        },
        "DecreaseMinimumPartDensityMode": {
          "Enum": 0
        },
        "StreamingEnabled": true
      },
      "$attributes": {
        "Rojo_Target_CurrentCamera": "62e89c49e4f800c20629c71b00003fcb"
      },
      "$path": "src"
    }
  }
}